ngraph/src/ngraph/opsets/opset4_tbl.hpp

```diff
@@ -153,4 +154,4 @@ NGRAPH_OP(TopK, ngraph::op::v3)
 
 // New operations added in opset4
 NGRAPH_OP(NonMaxSuppression, ngraph::op::v4)
-NGRAPH_OP(Acosh, ngraph::op::v3)
```

Comment:
Why did you remove this line?

Reply:
This line was moved to new place (in alphabet order).